archeologies-et-sciences-de-lantiquite.csl

```diff
@@ -0,0 +1,700 @@
+      <else-if type="manuscript" match="any">
+        <group>
+          <group suffix=",">
+            <text variable="title" text-case="capitalize-first" font-style="italic"/>
+          </group>
+          <text variable="genre" prefix=" "/>
+        </group>
+      </else-if>
+      <else-if type="entry-dictionary entry-encyclopedia chapter" match="any">
+        <group>
+          <group suffix=",">
+            <text variable="title" text-case="capitalize-first" quotes="true"/>
+          </group>
+          <text value="in" font-style="italic" suffix=" " prefix=" "/>
+          <text macro="editor" suffix=", "/>
+          <text variable="container-title" text-case="capitalize-first" font-style="italic"/>
+        </group>
+      </else-if>
+      <else-if type="paper-conference" match="any">
+        <group>
+          <group suffix=",">
+            <text variable="title" text-case="capitalize-first" quotes="true"/>
+          </group>
+          <text value="in" font-style="italic" prefix=" " suffix=": "/>
+          <text macro="editor" suffix=", "/>
+          <text variable="container-title" text-case="capitalize-first" font-style="italic"/>
+        </group>
+      </else-if>
+      <else-if type="post-weblog post webpage" match="any">
+        <group delimiter=", ">
+          <text variable="title" text-case="capitalize-first" quotes="true" font-style="normal"/>
+          <text variable="genre"/>
+          <text variable="container-title" font-style="italic"/>
+        </group>
+      </else-if>
+      <else-if type="map" match="any">
+        <group delimiter=", ">
+          <text variable="title" text-case="capitalize-first" font-style="italic"/>
+          <text variable="genre"/>
+          <text variable="scale"/>
+          <text variable="edition"/>
+        </group>
+      </else-if>
+      <else>
+        <text variable="title" text-case="capitalize-first" font-style="italic"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="disambiguate">
+    <choose>
+      <if disambiguate="true">
+        <choose>
+          <if type="book graphic report map personal_communication interview motion_picture thesis broadcast speech manuscript" match="any">
+            <text variable="title" form="short" font-style="italic"/>
+            <text term="cited" font-style="italic" prefix=", "/>
+          </if>
+          <else>
+            <text variable="title" text-case="capitalize-first" form="short" quotes="true" font-style="normal"/>
+            <text term="cited" font-style="italic" prefix=", "/>
+          </else>
+        </choose>
+      </if>
+      <else>
+        <text term="cited" font-style="italic" suffix="."/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="translator">
+    <text term="translator" form="verb" suffix=" "/>
+    <names variable="translator">
+      <name font-style="normal" and="text" delimiter-precedes-last="never" et-al-min="15" initialize-with="." sort-separator=" ">
```

Comment:
@avsz, every "et-al-min" attribute should be accompanied by an "et-al-use-first" attribute. ("et-al-min" specifies the minimum number of authors for which et-al abbreviation kicks in, and "et-al-use-first" specifies the number of authors that is still rendered before "et al.".